src/plugins_datastore.h

```diff
@@ -111,7 +111,7 @@ typedef sr_error_info_t *(*srds_conn_init)(sr_conn_ctx_t *conn, void **plg_data)
 typedef void (*srds_conn_destroy)(sr_conn_ctx_t *conn, void *plg_data);
 
 /**
- * @brief Store data for a module. Either a diff can be applied manually (if available) or full new data tree stored.
+ * @brief Store or prepare to store data for a module. Either a diff can be applied manually (if available) or full new data tree stored.
```

Comment:
Please add a new `typedef` so that every callback has its own, just like before. And mention the main difference in the docs, in the held locks. Also, in some use-cases the WRITE lock is held already for `store_prepare()`, so either fix that or mention it clearly (so that the callback never relies on it).